src/administrator/components/com_kunena/controllers/categories.php

```diff
@@ -441,11 +441,11 @@ protected function _save()
 			}
 
 			$success = $category->save();
-			$aliases = explode(',', JRequest::getVar('aliases_all'));
+			$aliases = explode(',', $app->input->getString('aliases_all'));
 
 			if ($aliases)
 			{
-				$aliases = array_diff($aliases, JRequest::getVar('aliases', array(), 'post', 'array'));
+				$aliases = array_diff($aliases, $app->input->post->getArray(array('aliases' => '')));
```

Comment:
yes, with jrequest its the same bug, so i will merge this one, but we need to look where the bug is